### DISCUSSION TITLE
Unable to modify controls since 0.24.1 version ... Bug or new feature?

### DISCUSSION THREAD

**Discussion by Michel7GitHub** [flet-dev/flet#4024] [Q&A]
### Question

I use 2 Windows environments, one with Flet version 0.23.2, the other one with Flet 0.24.1.
The following code works well on Flet 0.23.2 but doesn't work on 0.24.1.
I used few times that way to init & modify my controls

```
import flet as ft
def main(page: ft.Page):
    t1 = ft.TextField( label="Search 1" )
    t2 = ft.TextField( label="Search 2" )
    c1 = []
    cont = ft.Container(
        ft.Column( controls = c1 )
    )
    c1.append(t1)
    c1.append(t2)
    page.add( cont )
ft.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Muddassir-Farooq-official** [comment]
Use controls.append()

**Reply by Michel7GitHub** [reply]
Hi Muddassir,
`c1` is already a list in the `Column controls`. I can't use `c1.controls.append(t1)`. 
If you do, you will have the following message: `AttributeError: 'list' object has no attribute 'controls'`
Have you tried the code in both versions (0.23.2 & 0.24.1) ? 
You will see that the code I gave before works perfectly in version 0.23.2 but doesn't work in version 0.24.1
Regards,
Michel

**Reply by SUYinFu** [reply]
> Hi Muddassir, `c1` is already a list in the `Column controls`. I can't use `c1.controls.append(t1)`. If you do, you will have the following message: `AttributeError: 'list' object has no attribute 'controls'` Have you tried the code in both versions (0.23.2 & 0.24.1) ? You will see that the code I gave before works perfectly in version 0.23.2 but doesn't work in version 0.24.1 Regards, Michel

page.update()
